Package.swift

```diff
@@ -36,9 +36,9 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-parsing.git", from: "0.12.0"),
         .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.7.2"),
         .package(url: "https://github.com/scinfu/SwiftSoup.git", from: "2.3.2"),
-        .package(url: "https://github.com/vapor/fluent-postgres-driver.git", from: "2.0.0-rc"),
-        .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0-rc"),
-        .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0-rc"),
+        .package(url: "https://github.com/vapor/fluent-postgres-driver.git", exact: "2.6.2"),
```

Comment:
I've had to pin `fluent-postgres-driver` for now to fix a number of test failures (24 in total). I'm not sure if they're cause by the `fluent-postgres-driver` version bump or the underlying `postgres-kit` change - but pinning `fluent-postgres-driver` fixes it.